types/extra-types.d.ts

```diff
@@ -0,0 +1,163 @@
+/**
+ * This file contains the types that are required for compilation of the
+ * Polymer generated type declarations, but which could not themselves be
+ * automatically generated.
+ */
+
+/// <reference path="lib/mixins/property-effects.d.ts" />
+
+/**
+ * Types from "externs/closure-types.js"
+ */
+
+type Polymer_PropertyEffects = Polymer.PropertyEffects;
+
+/**
+ * Types from "externs/polymer-externs.js"
+ */
+
+interface PolymerElementPropertiesMeta {
+  type?: Function;
+  value?: any;
+  readOnly?: boolean;
+  computed?: string;
+  reflectToAttribute?: boolean;
+  notify?: boolean;
+  observer?: string;
+}
+
+type PolymerElementProperties = {
+  [key: string]: PolymerElementPropertiesMeta
+};
+
+interface PolymerInit {
+  is: string;
```

Comment:
Are there any comments to copy here? TODO maybe?

Reply:
Added TODO. Nothing in the externs to copy so I'd be writing something new.